src/cultionet/scripts/cultionet.py

```diff
@@ -1041,6 +1068,42 @@ def spatial_kfoldcv(args):
         )
 
 
+def generate_model_graph(args):
+    from cultionet.models.convstar import StarRNN
+    from cultionet.models.nunet import ResUNet3Psi
```

Comment:
was there a reason we import inside a function for this one?
Is it because the imports are only relevant for this function + take a long time to import?

Reply:
> Is it because the imports are only relevant for this function
This -- this function only serves the purpose of creating .onnx files for viewing graphs. It's called in isolation and there's no need for the imports elsewhere.